### ISSUE TITLE
Segmentation fault when deploying with docker-compose. 

### ISSUE THREAD

**Issue by Ornimi** [K4rian/docker-yuzu-room#3]
Hello, 
I encountered a segmentation fault while deploying with docker-compose. I use the "docker-compose.yml" by default in the github. 
![image](https://github.com/K4rian/docker-yuzu-room/assets/155017209/a883ab07-4a65-4604-af01-a87ea89389d9)

I dont know what is the issue.

**Comment by K4rian** [comment]
Hi,

Any chance to get more information about what the issue was in the first place, for the future users who might encounter the same error?

Thanks.

**Comment by sehraf** [comment]
I get the same on ARM64 when running the image directly.
When using `docker-compose` I get this error
```
> docker-compose up -d yuzu-room
[+] Running 1/2
 ✔ Network management_default  Created                                                                                                                                                                                                                                                                                   0.2s 
 ⠸ Container yuzu-room         Starting                                                                                                                                                                                                                                                                                  1.2s 
Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "./docker-entrypoint.sh": stat ./docker-entrypoint.sh: no such file or directory: unknown
```
No idea if this is related to the segfault.

**Comment by K4rian** [comment]
Hi @sehraf,

The OCI runtime error you're encountering might be related to a permission issue, especially since the image is working fine on amd64. Could you try running the image directly with Docker (without using Compose) to see if the issue persists?

Since this seems to be unrelated to the problem reported by @Ornimi, could you please open a new issue for it?

Thanks.

**Comment by sehraf** [comment]
Running `docker run --name yuzu-room -p 24872:24872/tcp -p 24872:24872/udp -i k4rian/yuzu-room` directly on ARM64 results in said `Segmentation fault (core dumped)`.

**Comment by K4rian** [comment]
Fixed in 4d3ab04.
